site/en/docs/privacy-sandbox/attribution-reporting-event-introduction/index.md

```diff
@@ -0,0 +1,321 @@
+
+```text
+Click 1298765678762 led to a conversion of value 1.
+```
+
+You may receive the following report:
+
+```text
+Click 1298765678762 led to a conversion of value 4.
+```
+
+### Report limitation
+
+To protect user privacy, the number of reports is limited. For a given source event (click), **at most three conversions can be registered**.
+
+{% Aside  %}
+This means that conversions will be dropped if a user converts more than three times. You can choose which conversions to get and which are fine to drop by configuring [conversion priorities](/docs/privacy-sandbox/attribution-reporting-event-guide/#prioritize-specific-conversions).
+{% endAside %}
+
+#### Example
+
+A user clicks an ad. Later on, they visit the advertiser site (first conversion), then
+create an account (second), and finally make two separate purchases (third and fourth).
+
+Even though they have converted four times, by default, you'll receive a report only for the first three conversions.
+
+### Report scheduling
+
+Reports are **not sent immediately after a user converts**. They're sent with a delay. This is a privacy protection.
+
+After the initial ad click, a schedule of **reporting windows** begins. Each reporting window has a
+deadline. Conversions registered before that deadline will be sent at the end of that window.
+
+{% Aside 'caution' %}
+Reports may not be exactly sent at the scheduled dates. If the browser isn't running
+when a report is scheduled to be sent, the report is sent at browser startup—which could be days or
+weeks after the initially scheduled time.
+{% endAside %}
+
+After expiry (click time + `attributionexpiry`), no additional attribution can be triggered (no additional conversion can be counted).
+
+#### Example
+
+If a user clicks an ad and converts three days later, you may receive the report about seven days
+later.
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+Report scheduling: detail
+{% endDetailsSummary %}
+
+In Chrome, report scheduling for clicks works as follows:
+
+<div class="w-table-wrapper"><table class="w-table--top-align"><thead><tr><th>Value of <code >attributionexpiry</code><br></th><th>Depending on the conversion time, a report is sent (if the browser is open)...</th><th>Number of reporting windows</th></tr></thead><tbody><tr><td>30 days (the default and maximum value)</td><td><ul><li>2 days after the ad was clicked</li><li>or 7 days after click</li><li>or ((<code>attributionexpiry</code>= 30 days) + 1 hour) after click.</li></ul></td><td>3</td></tr><tr><td>Between 7 and 30 days</td><td><ul><li>2 days after click</li><li>or 7 days after click</li><li>or (<code>attributionexpiry</code> + 1 hour) after click.</li></ul></td><td>3</td></tr><tr><td>Between 2 and 7 days</td><td><ul><li>2 days after click</li><li>or (<code>attributionexpiry</code> + 1 hour) after click.</li></ul></td><td>2</td></tr><tr><td>Under 2 days</td><td><ul><li>(2 days + 1 hour) after click</li></ul></td><td>1</td></tr></tbody></table></div>
```

Comment:
```suggestion
<div class="w-table-wrapper">
    <table class="w-table--top-align">
        <thead>
            <tr>
                <th>Value of <code>attributionexpiry</code><br></th>
                <th>Time a report is sent, depending on the conversion time and whether the browser is open</th>
                <th>Number of reporting windows</th>
            </tr>
        </thead>
        <tbody>
            <tr>
                <td>30 days (the default and maximum value)</td>
                <td>
                    <ul>
                        <li>2 days after the click</li>
                        <li>or 7 days after the click</li>
                        <li>or at maximum, 30 days + 1 hour after the click</li>
                    </ul>
                </td>
                <td>3</td>
            </tr>
            <tr>
                <td>Between 7 and 30 days</td>
                <td>
                    <ul>
                        <li>2 days after the click</li>
                        <li>or 7 days after the click</li>
                        <li>or the value of <code>attributionexpiry</code + 1 hour after the click</li>
                    </ul>
                </td>
                <td>3</td>
            </tr>
            <tr>
                <td>Between 2 and 7 days</td>
                <td>
                    <ul>
                        <li>2 days after click</li>
                        <li>or at maximum, the value of <code>attributionexpiry</code> + 1 hour after the click</li>
                    </ul>
                </td>
                <td>2</td>
            </tr>
            <tr>
                <td>Under 2 days</td>
                <td>
                    <ul>
                        <li>2 days + 1 hour after the click</li>
                    </ul>
                </td>
                <td>1</td>
            </tr>
        </tbody>
    </table>
</div>
```

Reply:
I changed the wording of most rows here, look closer to verify it's alright

Reply:
Ty, I've used your table + made a small tweak (grey header).